docs/design/README.md

```diff
@@ -539,15 +539,21 @@ the array may be deduced, it may be omitted, as in:
 
 ```carbon
 var i: i32 = 1;
+// [i32;N] [i32;N] is an array representing N values, the numerical type of each value is the data type of i32, the size of the array is determined, and new elements cannot be added, nor can the elements of the array be reduced.
+   var a:[i32;5] = (2,4,5,7,9);
 // `[i32;]` equivalent to `[i32; 3]` here.
-var a: [i32;] = (i, i, i);
+var a: [i32;] = (i,j,k);
```

Comment:
`j` and `k` haven't been declared yet. Maybe we should change this to something like:

```suggestion
var a: [i32;] = (1, 2, 3);
```

Reply:
var a:[i32;] = (i,j,k) It means that it contains three elements, which can be the same value or different values. If a:[i32;] =(1,2,3) is more straightforward, I can modify it again to submit in this way.

Reply:
Yes, I think it would be more straightforward to avoid mentioning variables that aren't declared in the example.

Reply:
> var a:[i32;] = (i,j,k) It means that it contains three elements, which can be the same value or different values. If a:[i32;] =(1,2,3) is more straightforward, I can modify it again to submit in this way.

Reply:
> 是的，我认为避免提及示例中未声明的变量会更直接。

Then I will rewrite this var:a[i,j,k] to var a:[1,3,5] to modify this document?

Reply:
I don't understand; there's no "var:a[i,j,k]" in this document right now.

Please see the suggested edit in my first comment above.

Reply:
> I don't understand; there's no "var:a[i,j,k]" in this document right now.
> 
> Please see the suggested edit in my first comment above.

ok i got it, i read the docs, i don't know where else i need to contribute the code, i want to contribute to this, if possible i want to write the front end code for the array and slice part, or other parts too Can.